cdap-ui/app/cdap/components/PipelineConfigurations/PipelineConfigurations.scss

```diff
@@ -0,0 +1,459 @@
+          line-height: 20px;
+        }
+
+        .btn .icon-close {
+          color: $text-color;
+          width: 16px;
+          height: 16px;
+        }
+      }
+
+      .pipeline-configurations-body {
+        display: flex;
+
+        /* 60px: height of modeless box header */
+        height: calc(100% - 60px);
+
+        .configurations-side-panel {
+          border-right: 1px solid $border-color;
+          border-top: 1px solid $border-color;
+
+          .configurations-tabs {
+            .configuration-tab {
+              height: 40px;
+              display: flex;
+              align-items: center;
+              padding-left: 23px;
+              font-size: $regular-font;
+              cursor: pointer;
+              width: $side-panel-width;
+              border-bottom: 1px solid $border-color;
+
+              &:hover {
+                color: $cdap-orange;
+              }
+
+              &.active {
+                background-color: $active-bg-color;
+                color: $cdap-orange;
+                font-weight: bold;
+              }
+
+              &.disabled {
+                pointer-events: none;
+                color: lighten($text-color, 60%);
+              }
+
+              &.toggle-advanced-options {
+                padding-left: 0;
+
+                .icon-caret-right,
+                .icon-caret-down {
+                  height: 14px;
+                  width: 14px;
+                  margin: 0 1px 0 9px;
+                }
+              }
+            }
+          }
+        }
+
+        .configuration-content {
+          width: 100%;
+          margin: 16px 30px 40px 50px;
+          font-size: $regular-font;
+          position: relative;
+
+          .configuration-content-container {
+            /* 50px: height from bottom of buttons to horizontal line */
+            height: calc(100% - 50px);
+            border-bottom: 1px solid $border-color;
+
+            .configuration-step-content {
+              height: 100%;
+            }
+          }
+
+          .configuration-step-content {
+            .step-content-heading {
+              font-weight: bold;
+              margin-bottom: 16px;
+
+              .step-content-subtitle {
+                font-weight: initial;
+                font-size: 12px;
+              }
+            }
+
+            .label-with-toggle {
+              margin-bottom: 15px;
+              display: flex;
+              align-items: center;
+
+              .toggle-label {
+                text-align: right;
+                padding-right: 0;
+              }
+
+              .toggle-container {
+                display: flex;
+                align-items: center;
+
+                .icon-info-circle {
+                  margin-left: 10px;
+                }
+              }
+
+              .small-dropdown {
+                width: 65px;
+                display: inline-block;
+              }
+            }
+
+            .key-value-pair-labels {
```

Comment:
Looks like it could be in a component? Just trying to see if we can split the styles to individual components so that this file can be reduced to simpler ones.